operator/pkg/test/environment/environment.go

```diff
@@ -55,8 +55,9 @@ func (e *Environment) Stop() error {
 	return e.Environment.Stop()
 }
 
-func crdFilePath() string {
+func crdFilePaths() []string {
 	_, file, _, _ := runtime.Caller(0)
 	p := filepath.Join(filepath.Dir(file), "..", "..", "..")
-	return filepath.Join(p, "charts/kit-operator/crds/control-plane-crd.yaml")
+	// includes CRDs for podmonitors
+	return []string{filepath.Join(p, "charts/kit-operator/crds/control-plane-crd.yaml"), filepath.Dir(file)}
```

Comment:
It should work with just the folder, I think.

Reply:
Yeah the `filepath.Dir(file)` just passes the directory. In case of `charts/kit-operator/crds/control-plane-crd.yaml` we are only installing control-plane CRD and skipping the Dataplane CRD.